### PR TITLE
doc: Update instructions for writing an SD card

### PR DESCRIPTION
**Pull request by dNechita**
Addresses #196

### CODE REVIEW

File: doc/sdcard_burn.md [hunk 1]
```diff
@@ -1,26 +1,34 @@
...
     ~~~
 * Write the image on the SD card
     ~~~sh
-    dd if=xxx.img of=/dev/xxx bs=4M
+    dd if=name-of-the-image.img of=/dev/xxx bs=4M
     sync
     ~~~
     
-Note: The **lsblk** command can be used to help identifying the device to be specified for the argument: **of=/dev/xxx**.
+Note 1: The following command can be used to help identifying the device than needs to be specified for the argument **of=/dev/xxx**:
```
Comment (grigorovici-andreea) [inline doc/sdcard_burn.md]: I think here should be **that**: "....the device than needs to be specified...."

File: doc/sdcard_burn.md [hunk 2]
```diff
@@ -1,26 +1,34 @@
...
+~~~sh
+    lsblk
+~~~
+Note 2: It is very likely that the **dd** command will have to be executed with **sudo** priviledges:
```
Comment (grigorovici-andreea) [inline doc/sdcard_burn.md]: here is: **privileges**

File: doc/sdcard_burn.md (hunk 1 above)
Comment (dNechita) [reply doc/sdcard_burn.md]: yes, fixed. Thanks!

File: doc/sdcard_burn.md (hunk 2 above)
Comment (dNechita) [reply doc/sdcard_burn.md]: it is. Fixed.

**Pull request merged**